### ISSUE TITLE
Keyboard navigation on browser

### ISSUE THREAD

**Issue by mycroft89** [patelneeraj/filebrowserplus.koplugin#39]
Pretty please, implement some keyboard navigation in the browser. Whether I'm missing something, or pressing tab on my keyboard doesn't do anything. :(

**Comment by patelneeraj** [comment]
@mycroft89 I am not the author of [filebrowser/filebrowser](https://github.com/filebrowser/filebrowser). This, [filebrowserplus.koplugin](https://github.com/patelneeraj/filebrowserplus.koplugin) is a separate plugin written to bring the functionality provided by [filebrowser/filebrowser](https://github.com/filebrowser/filebrowser) to KOReader. The browser UI that you see is provided by [filebrowser/filebrowser](https://github.com/filebrowser/filebrowser) and thus I cannot implement the requested feature.

**Comment by mycroft89** [comment]
> [@mycroft89](https://github.com/mycroft89) I am not the author of [filebrowser/filebrowser](https://github.com/filebrowser/filebrowser). This, [filebrowserplus.koplugin](https://github.com/patelneeraj/filebrowserplus.koplugin) is a separate plugin written to bring the functionality provided by [filebrowser/filebrowser](https://github.com/filebrowser/filebrowser) to KOReader. The browser UI that you see is provided by [filebrowser/filebrowser](https://github.com/filebrowser/filebrowser) and thus I cannot implement the requested feature.

Thanks for your detailed response anyway. I've just learned that the original plugin is about to [die for real](https://hacdias.com/2026/07/28/filebrowser/) pretty soon. I guess that will mark the end of filebrowserplus too?

**Comment by patelneeraj** [comment]
Yes, please see my comment #32
